podcasts/TranscriptFilter.swift

```diff
@@ -49,9 +60,23 @@ extension RegexFilter {
     // Remove SRT tags, for example: "Speaker 1: " to ""
     static let speakerFilter = RegexFilter(pattern: "Speaker \\d?: *", replacement: "")
     // Ensure that any full stop starts a new line
-    static let fullStopNewLineFilter = RegexFilter(pattern: "([\\!\\?\\.])\\s*", replacement: "$1\n")
+    static let fullStopNewLineFilter = RegexFilter(pattern: "([\\!\\?\\.])\\s+", replacement: "$1\n")
+    // Full Stop at end of cue
+    static let fullStopEndofCueFilter = RegexFilter(pattern: "([\\!\\?\\.])\\z", replacement: "$1\n")
     // Ensure that end of cues have a space when appended to the next cue
     static let notfullStopNewLineFilter = RegexFilter(pattern: "([^\\!\\?\\.])\\z", replacement: "$1 ")
+    // &nbsp filter
+    static let nbspFilter = RegexFilter(pattern: "&nbsp;", replacement: " ")
+    // <br> filter
+    static let breakLineFilter = RegexFilter(pattern: "<br>|<BR>|<br/>|<BR/>", replacement: "\n")
```

Comment:
This doesn't work with `<br />`, which is a valid tag

Reply:
Will add to the list.